docs/src/lib/operations.md

```diff
@@ -449,3 +449,9 @@ to_union_of_projections(::Rectification{N}, ::Bool=false) where {N<:Real}
 ```@docs
 RectificationCache
 ```
+
+## Reflection
+
+```@docs
+reflect(::LazySet)
```

Comment:
Just a comment: We currently only list lazy operations in this file (sorry, I did not see this before). This may change in the future, but for now we add these docs in the file `interfaces.md`.